projects/layout/components/app-bar/app-bar-back.style.less

```diff
@@ -7,6 +7,7 @@
     align-items: center;
     padding: 0 0.625rem 0 0.125rem;
     cursor: pointer;
+    font-size: 0;
```

Comment:
It supports text on iOS and Web, that's why `font-size: 0;` is only for Android.

Reply:
fixed